docs/concepts/qlight-node.md

```diff
@@ -1,7 +1,7 @@
 # GoQuorum qlight node
```

Comment:
The [navigation entry is "GoQuorum qlight client"](https://github.com/ConsenSys/doc.goquorum/pull/212/files#diff-846dcbef7a75e6dd527f651c8472b00ff26082c0ce9b251efa1407c6600bcdd5R96), you may make it consistent in the page content (either name the nav entry "node" or "node and client" or maybe just "qlight" and no mention of node and client in the titles as you have the sections in the content for each of them)